src/portable/windows.rs

```diff
@@ -747,7 +750,7 @@ pub fn list_versions(options: &options::ListVersions) -> anyhow::Result<()> {
     } else {
         log::warn!(
             "WSL distribution is not installed, \
-                   so no EdgeDB server versions are present."
+                   so no {BRANDING} server versions are present."
```

Comment:
Maybe we shouldn't duplicate this one message 10 times?

Reply:
Doesn't really need to be fixed now though

Reply:
I'm hoping to fix this in a later PR whilest fixing the duplicated print functions but totally agree.